src/main/resources/assets/gregtechfoodoption/lang/en_us.lang

```diff
@@ -379,6 +379,19 @@ metaitem.solidified_mozzarella_curd_fuel_pure.name=Solidified Mozzarella Curd
 metaitem.dried_mozzarella_curd_fuel_pure.name=Dried Mozzarella Curd
 metaitem.small_mozzarella_curd_fuel_pure.name=Small Mozzarella Curd
 metaitem.large_mozzarella_curd_fuel_pure.name=Large Mozzarella Curd
+metaitem.gorgonzola_curd_fuel_pure.name=Gorgonzola Curd
+metaitem.penicilinium_roqueforti_dust.name=Penicilinium Roqueforti
+fluid.fungal_rennet_solution=Crude Rennet Solution
```

Comment:
Why is this particularly named "Crude Rennet Solution"? I would think it would be named "Fungal Rennet Solution". Also, as you may have noticed, I generally prefer putting a space between each new update's content, so I would recommend that you do that.

Reply:
The trap of Ctrl+C, Ctrl+V